src/react.js

```diff
@@ -20,4 +20,20 @@ module.exports = {
     'react/react-in-jsx-scope': 0,
     'react/jsx-uses-react': 0,
   },
+  overrides: [
+    {
+      files: ['**/components/**/*.tsx', '**/components/**/*.jsx'],
```

Comment:
I think we can just set the wildcard for specified ext `*.tsx` and `*.jsx` and The container directory should not be part of the lint rule. WDYT?

Reply:
will that work for the nextjs pages? They need to be in kebab-case

Reply:
Can we make override rules for the `pages` dir?

Reply:
ah yeah it works because I overwrite it again for the files in the pages directory in the nextjs file